roles/lacchain-writer-node/templates/pantheon-config.j2

```diff
@@ -5,7 +5,7 @@ rpc-ws-enabled=true
 rpc-http-port=4545
 graphql-http-port=4547
 rpc-ws-port=4546
-rpc-http-api=["ETH","NET","IBFT","EEA","PRIV"]
+rpc-http-api=["ETH","NET","IBFT","EEA","PRIV","ADMIN"]
```

Comment:
It looks a bit dangerous to enable ADMIN by default.

Reply:
solved @davux , thanks